pkg/framework/context/context.go

```diff
@@ -113,7 +113,7 @@ func (p *Provider) InboundMessageHandler() transport.InboundMessageHandler {
 		// find the service which accepts the message type
 		for _, svc := range p.services {
 			if svc.Accept(msg.Header.Type) {
-				_, err = svc.HandleInbound(msg)
+				_, err = svc.HandleInbound(msg, "", "")
```

Comment:
Should `transport.InboundMessageHandler` be refactored as well?

Reply:
It should be done here https://github.com/hyperledger/aries-framework-go/pull/938
Also, all files which are not related to the introduce protocol will be reverted according to PR above.

Reply:
Except for changes that are related to `HandleOutbound` function. I have changed `HandleOutbound` function signature.

Reply:
@Moopli  Could you please confirm?

Reply:
found! https://github.com/hyperledger/aries-framework-go/pull/938/files#diff-9988f5820bf05228e5657f2842fa071eR114

Reply:
Given how you're using HandleOutbound, it seems like a reasonable change to the interface.

HandleOutbound isn't used in DIDExchange at all, so we can make it use DIDs, which makes things easier for other protocols.